azureadps-2.0/AzureAD/Set-AzureADMSNamedLocationPolicy.yml

```
@@ -17,13 +17,13 @@ syntaxes:
 examples:
 - title: 'Example 1: Update an ip named location policy in Azure AD by PolicyId.'
   code: |-
-    PS C:\> Set-AzureADMSNamedLocationPolicy -PolicyId 07a1f48d-0cbb-4c2c-8ea2-1ea00e3eb3b6 -OdataType "#microsoft.graph.ipNamedLocation" -IsTrusted $false
+    PS C:\> Set-AzureADMSNamedLocationPolicy -PolicyId 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5 -OdataType "#microsoft.graph.ipNamedLocation" -IsTrusted $false
```

Policy id now excluded from sensitive term list
```suggestion
    PS C:\> Set-AzureADMSNamedLocationPolicy -PolicyId 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5 -OdataType "#microsoft.graph.ipNamedLocation" -IsTrusted $false
```